src/python/foglamp/postgres_handler.py

```diff
@@ -0,0 +1,82 @@
+import time
+import logging
+import logging.handlers
+import asyncio
+import sqlalchemy as sa
+import aiopg.sa
+import psycopg2
+from foglamp.configurator import Configurator
+from aiopg.sa import create_engine
+
+metadata = sa.MetaData()
+
+_log_tbl = sa.Table(
+    'log_t'
+    , metadata
+    , sa.Column('log_level', sa.types.INT)
+    , sa.Column('log_levelname', sa.types.VARCHAR(10))
+    , sa.Column('log', sa.types.VARCHAR(100))
+    , sa.Column('created_at', sa.types.DATE)
+    , sa.Column('created_by', sa.types.VARCHAR(50)))
+
+def log_msg(record):
+    """Construct log message from logging record components"""
+
+    msg = record.msg
+    msg = msg.strip()
+    msg = msg.replace('\'', '\'\'')
+
+    return msg
```

Comment:
All postgres access should be moved to a module in foglamp/model package

See https://scaledb.atlassian.net/browse/FOGL-128